extensions/notebook/src/test/common.ts

```diff
@@ -20,6 +20,10 @@ export class JupyterServerInstanceStub implements IServerInstance {
 		return undefined;
 	}
 
+	public get attachedEditorCount(): number {
```

Comment:
looks like tslint is not turned on for this extension? it will cause an error like: undefined cannot be assigned to number.

Reply:
Yes, the no strict nulls checks are only on (parts) of the core code for now. It would be great at some point to get the extensions running under that too but that's going to be a whole lot of refactoring/fixing.

In the meantime it's good practice to get in the habit of returning the correct types though. In this case returning something like -1 seems appropriate. 

---
In reply to: [335589344](https://github.com/microsoft/azuredatastudio/pull/7744#discussion_r335589344) [](ancestors = 335589344)